### ISSUE TITLE
tomlib: fusion problem for group 2.(A4xA4).2.2

### ISSUE THREAD

**Issue by vdlee37** [gap-system/gap#3595]
┌───────┐   GAP 4.10.2 of 19-Jun-2019
     │  GAP  │   https://www.gap-system.org
     └───────┘   Architecture: i686-pc-cygwin-default32-kv3
    
    gap> t:=TableOfMarks("2.(A4xA4).2.2");
    TableOfMarks( "2.(A4xA4).2.2" )
    
    gap> c:=CharacterTable("2.(A4xA4).2.2");
    fail
    
    gap> g:=UnderlyingGroup(t);
    Group([ (1,11,9,5)(2,7,3,6)(4,12,10,8)(13,16,15,14)(17,23,22,18)(19,24)(20,21), (1,4,3,2)(5,14,12,6)(8,11)(9,15,13,10)
    (17,19,20,23)(18,22,24,21), (1,15,13,9,10,3,2,4)(5,21,11,22,12,20,8,17)(6,19,7,23,14,24,16,18) ])
    
    gap> c:=CharacterTable(g);
    CharacterTable( Group([ (1,11,9,5)(2,7,3,6)(4,12,10,8)(13,16,15,14)(17,23,22,18)(19,24)(20,21), (1,4,3,2)(5,14,12,6)
    (8,11)(9,15,13,10)(17,19,20,23)(18,22,24,21), (1,15,13,9,10,3,2,4)(5,21,11,22,12,20,8,17)(6,19,7,23,14,24,16,18) ]) )
    
    gap> f:=FusionCharTableTom(c,t);
    [ 1, 2, 4, 25, 9, 38, 11, 40, 108, 48, 10, 32, 18, 76, 3, 7, 8, 14, 44, 50, 5, 6, 13, 39, fail ]
                                                                                              ====

Notice that the final entry has value FAIL.

**Comment by ThomasBreuer** [comment]
@vdlee37 Thanks for this bug report.
The reason for the problem seems to be that `RepresentativeTom` does not return the correct representatives.
```
gap> t:= TableOfMarks( "2.(A4xA4).2.2" );;
gap> g:= UnderlyingGroup( t );;
gap> IsConjugate( g, RepresentativeTom( t, 39 ), RepresentativeTom( t, 43 ) );
true
```
I have opened an issue for that in the issue tracker of the `TomLib` package.

Concerning the original idea to use the table of marks and the character table from the GAP databases, the names of corresponding tables are not necessary equal, but you can do the following.
```
gap> t:= TableOfMarks( "2.(A4xA4).2.2" );
TableOfMarks( "2.(A4xA4).2.2" )
gap> c:= CharacterTable( t );
CharacterTable( "w(f4)" )
gap> f:=FusionCharTableTom(c,t);
[ 1, 2, 4, 25, 11, 40, 9, 38, 10, 48, 108, 3, 18, 32, 76, 5, 6, 8, 7, 13, 14, 
  43, 39, 44, 50 ]
```

**Comment by DominikBernhardt** [comment]
@ThomasBreuer  As you have transfered this issue to the Issue tracker of the  `TomLib` package,
can this Issue be closed?

**Comment by ThomasBreuer** [comment]
@DominikBernhardt I think it depends. If one regards a problem as solved as soon as it has been reported to some responsible person then yes. If one regards a problem as solved as soon as it disappears then no.

**Comment by DominikBernhardt** [comment]
@ThomasBreuer I thought our policy was to close Issues if they are moved to a package (and belong there). Maybe I was mistaken.

**Comment by fingolfin** [comment]
Fixed in https://github.com/gap-packages/tomlib/pull/28, which needs to be merged and then tomlib needs a new release

**Comment by vdlee37** [comment]
Thanks everyone!

**Comment by fingolfin** [comment]
A fixed tomlib was released, and will be shipped with GAP 4.11
